lib/config.ts

```diff
@@ -356,8 +356,18 @@ export interface Config {
   baseUrl?: string;
 
   /**
-   * CSS Selector for the element housing the angular app - this defaults to
-   * 'body', but is necessary if ng-app is on a descendant of <body>.
+   * A CSS selector for the DOM element you want Protractor to hook into your
+   * app at.
```

Comment:
Maybe

"A CSS Selector for a DOM element within your Angular application. Protractor will attempt to automatically find your application, but It is necessary to set rootElement in certain cases.

In Angular 1, Protractor will use bootstrapping and then search for `body` or `ng-app` elements (details here: https://git.io/v1b2r).

In later versions, Protractor will try to hook into all angular apps on the page. Use rootElement to limit the scope of which apps Protractor waits for and searches within.